modules/nf-core/oncocnv/main.nf

```diff
@@ -0,0 +1,73 @@
+process ONCOCNV {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::oncocnv=7.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://containers.biocontainers.pro/s3/SingImgsRepo/oncocnv/v7.0_cv2/oncocnv_v7.0_cv2.sif':
+        'registry.hub.docker.com/biocontainers/oncocnv:v7.0_cv2' }"
+
```

Comment:
sorry about this again: are the containers available via quay.io/galaxy project (where we normally get all our contaienrs from) ?